### ISSUE TITLE
Feature request: stages in MIP from cg directly

### ISSUE THREAD

**Issue by karlnyr** [Clinical-Genomics/cg#720]
**Background**
When the MIP pipeline fails in a certain stage you are able to use the -sw, --start-with, flag to start from a specified stage in the pipeline. This feature is very handy, as you are not required to rerun the heavier workload tasks. However, this requires the user to know the specific stage name.

**Feature request** 
The current version of cg does not allow the user to view get the workflow steps available. Instead, the user now has to activate another environment (the mip environment) and then request the list by issuing the command:

```shell
[user@ws] [P_main] $ source activate P_mip8.2_rd-dna
[user@ws] $ mip analyse rd_dna --help

<Insert long list of work stages here>
```

If this could be requested through cg instead it would be more intuitive to use the -sw flag.

For example:
```shell
[user@ws] [P_main] $ cg workflow mip-dna --stages

<Insert long list of work stages here>
```

**Comment by henrikstranneheim** [comment]
@karlnyr I wonder if it would be enough to add something like this in the docs for MIP:
```
use-mip && mip analyse rd_dna --help | grep "recipe switch" && down
```
We can also link to the initiation map in the docs at https://github.com/Clinical-Genomics/MIP/blob/master/definitions/rd_dna_initiation_map.yaml, which will tell you all the steps and the order of processing

**Comment by karlnyr** [comment]
I want to recall that the instructions of mip already covers how to get the possible stages:
```shell
up

use-mip

mip analyse <pipeline> --pr
```

**Comment by henrikstranneheim** [comment]
Yes, but I think you also need to provide a config and a case_id if you want to use that command

**Comment by karlnyr** [comment]
Oh, then I agree that the documentation could be updated 👍
